components/Navigation.js

```diff
@@ -79,6 +79,15 @@ const List = styled.nav`
             }
         }
     }
+    @media (max-width: 600px) {
```

Comment:
Wouldn't it be better to differ by height here? Width shouldn't be a problem in most cases...